src/cryptoadvance/specter/specter.py

```diff
@@ -234,6 +234,7 @@ def cleanup_on_exit(self, signum=0, frame=0):
             self._bitcoind.stop_bitcoind()
 
         logger.info("Closing Specter after cleanup")
+        raise SpecterError("Specter closed")
```

Comment:
What's the purpose of this? Maybe you can add a comment here, looks very unconventional.